fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -91,6 +91,7 @@
 import static com.apple.foundationdb.record.metadata.Key.Expressions.function;
 import static com.apple.foundationdb.record.provider.foundationdb.indexes.TextIndexTestUtils.COMPLEX_DOC;
 import static com.apple.foundationdb.record.provider.foundationdb.indexes.TextIndexTestUtils.SIMPLE_DOC;
+import static com.apple.foundationdb.record.provider.foundationdb.indexes.TextIndexTestUtils.SIMPLE_DOCUMENT_WITH_TEXT_PRIMARY_KEY;
```

Comment:
I guess it's good to have a regression test with a definitely non-fixed point primary key, but do any of the tests actually use the new format? I think they'd all fail the "is fixed width" check because they all have optional fields as their primary key